camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -938,8 +939,45 @@ def pt2line(x0, y0, x1, y1, x2, y2):
         else:
             return None
 
+    def create_mask(self, screen):
+        """
+        Create a full binary mask with the image shape
 
-    def handle_msg(self, msg):
+        :param screen: display
+        :type screen: :class:`cvMat`
+        """
+        shape = screen.shape
+        # Create the binary mask
+        self.mask = numpy.full((shape[0], shape[1]), 1, dtype=numpy.uint8)
```

Comment:
I dont see a reason to  use uint8 as a dtype.  Its used as a binary mask  and even converted to bool. just use bool